jerry-core/jrt/jrt.h

```diff
@@ -40,6 +40,18 @@
 # define __attr_pure___ __attribute__((pure))
 #endif /* !__attr_pure___ */
 
+#ifdef CONFIG_STORE_CONST_IN_ROM
+#ifndef ICACHE_RODATA_ATTR
+#define ICACHE_RODATA_ATTR  __attribute__((section(".irom.text")))
```

Comment:
This attribute seems ESP specific. It would be great if this would be defined by the build system.